db/seeds.rb

```diff
@@ -30,7 +32,15 @@
   description: "A marvelous marketplace for magic merchandise.",
   hero_image: FactoryBot.create(:media)
 )
-FactoryBot.create(:marketplace, :full, room: marketplace_section)
+marketplace = FactoryBot.create(:marketplace, :ready_for_shopping, product_quantity: 16, room: marketplace_section)
```

Comment:
I think this code exists in `main`, and should not be appearing as a diff in this PR.

Reply:
Ah, it's not actually. You made these changes in this branch with https://github.com/zinc-collective/convene/pull/2361/commits/cd5cd4b1e849fb37bf0384691b97e8f2cf2a4b37. ☺️

Reply:
Ah, it's not actually. You made these changes in this branch with https://github.com/zinc-collective/convene/commit/cd5cd4b1e849fb37bf0384691b97e8f2cf2a4b37. ☺️